### PR TITLE
Project Rebalance Achievement Diary Agility Shortcut Changes

### PR DESCRIPTION
**Pull request by shookietea**
https://secure.runescape.com/m=news/project-rebalance-skilling--poll-81-mta-changes?oldschool=1

Achievement Diary requirements were removed from some agility shortcuts. This PR brings the achievement diaries up to date.

### CODE REVIEW

File: src/main/java/com/questhelper/helpers/achievementdiaries/desert/DesertHard.java [hunk 1]
```diff
@@ -313,8 +313,7 @@ public List<UnlockReward> getUnlockRewards()
 	{
 		return Arrays.asList(
 			new UnlockReward("Pharaoh's sceptre can hold up to 50 charges"),
-			new UnlockReward("Access to the big window shortcut in Al Kharid Palace that takes you to the south of the palace, " +
-				"just north of the Shantay Pass, requiring 70 Agility"),
+			new UnlockReward("Access to the shortcut just north of the Shantay Pass, requiring 70 Agility"),
```
Comment (pajlada) [inline src/main/java/com/questhelper/helpers/achievementdiaries/desert/DesertHard.java:316]: This reward was removed outright, so the entry should just be removed

File: src/main/java/com/questhelper/helpers/achievementdiaries/karamja/KaramjaMedium.java [hunk 2]
```diff
@@ -393,8 +393,7 @@ public List<UnlockReward> getUnlockRewards()
 	{
 		return Arrays.asList(
 			new UnlockReward("Increased Agility Experience when redeeming Agility tickets"),
-			new UnlockReward("10% increased Agility experience earned from Brimhaven Agility Arena"),
-			new UnlockReward("Access to the stepping stone shortcut across the river of Shilo Village"));
+			new UnlockReward("10% increased Agility experience earned from Brimhaven Agility Arena"));
```
Comment (pajlada) [inline src/main/java/com/questhelper/helpers/achievementdiaries/karamja/KaramjaMedium.java:396]:
```suggestion
			new UnlockReward("10% increased Agility experience earned from Brimhaven Agility Arena when wearing Karamja gloves"));
```

**Review by pajlada — CHANGES_REQUESTED** [review]
Some nits + some additions you can add, if you want to, while you're in these files

Missing reward at the bottom of Desert Hard diary: `Increased experience on the Pollnivneach Rooftop Course`
Missing reward at the bottom of Fremennik Hard: `Increased experience on the Rellekka Rooftop Course`
Missing reward at the bottom of Karamja Medium: `10% additional Agility experience earned from all obstacles in the Brimhaven Agility Arena when wearing Karamja gloves`

**Comment by shookietea** [comment]
Happy to make those changes! For the Rellekka and Pollnivneach courses, is it bonus experience for completing laps, or each obstacle?

https://oldschool.runescape.wiki/w/Fremennik_Diary#Rewards_3 makes me wonder if I should be more precise with the wording, but I unfortunately cannot test these

**Comment by pajlada** [comment]
> https://oldschool.runescape.wiki/w/Fremennik_Diary#Rewards_3 makes me wonder if I should be more precise with the wording, but I unfortunately cannot test these

If you use the same phrasing the wiki team does, you should be fine

**Comment by shookietea** [comment]
> > https://oldschool.runescape.wiki/w/Fremennik_Diary#Rewards_3 makes me wonder if I should be more precise with the wording, but I unfortunately cannot test these
> 
> If you use the same phrasing the wiki team does, you should be fine

This is updated now, thanks!

**Review by pajlada — APPROVED** [review]

**Pull request merged**